src/class_inline.h

```diff
@@ -0,0 +1,19 @@
+static inline struct RProc *
+mrb_method_search_vm_proc(mrb_state *mrb, struct RProc *p, struct RClass **cp, mrb_sym mid)
```

Comment:
It just seems like an anti-pattern for mruby's current code organization, since we usually put headers in `include` if they're meant to be shared.